docs/user/resources/01-serverless.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Serverless
+---
+
+The `serverlesses.operator.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the Serverless configuration that you want to install on your cluster. To get the up-to-date CRD and show the output in the YAML format, run this command:
+
+   ```bash
+   kubectl get crd serverlesses.operator.kyma-project.io -o yaml
+   ```
+
+## Sample custom resource
+
+The following Serverless custom resource (CR) shows configuration of Serverless with the external registry and custom endpoints for eventing and tracing.
+
+   ```yaml
+   apiVersion: operator.kyma-project.io/v1alpha1
+   kind: Serverless
+   metadata:
+     finalizers:
+     - serverless-operator.kyma-project.io/deletion-hook
+     name: default
+     namespace: kyma-system
+   spec:
+     dockerRegistry:
+       enableInternal: false
+       secretName: my-secret
+     eventing:
+        endpoint: http://eventing-publisher-proxy.kyma-system.svc.cluster.local/publish
+     tracing:
+        endpoint: http://telemetry-otlp-traces.kyma-system.svc.cluster.local:4318/v1/traces
+   status:
+     conditions:
+     - lastTransitionTime: "2023-04-28T10:09:37Z"
+       message: Configured with default Publisher Proxy URL and default Trace Collector
+         URL.
+       reason: Configured
+       status: "True"
+       type: Configured
+     - lastTransitionTime: "2023-04-28T10:15:15Z"
+       message: Serverless installed
+       reason: Installed
+       status: "True"
+       type: Installed
+     eventPublisherProxyURL: http://eventing-publisher-proxy.kyma-system.svc.cluster.local/publish
+     state: Ready
+     traceCollectorURL: http://telemetry-otlp-traces.kyma-system.svc.cluster.local:4318/v1/traces
+   ```
+
+## Custom resource parameters
+<!-- TABLE-START -->
+### Serverless.operator.kyma-project.io/v1alpha1
+
+**Spec:**
+
+| Parameter | Type | Description |
+| ---- | ----------- | ---- |
+| **dockerRegistry**  | object |  |
+| **dockerRegistry.&#x200b;enableInternal**  | boolean |  |
+| **dockerRegistry.&#x200b;secretName**  | string |  |
+| **eventing**  | object |  |
+| **eventing.&#x200b;endpoint** (required) | string |  |
+| **tracing**  | object |  |
+| **tracing.&#x200b;endpoint** (required) | string |  |
+
+**Status:**
+
+| Parameter | Type | Description |
+| ---- | ----------- | ---- |
+| **conditions**  | \[\]object | Conditions associated with CustomStatus. |
+| **conditions.&#x200b;lastTransitionTime** (required) | string | lastTransitionTime is the last time the condition transitioned from one status to another. This should be when the underlying condition changed.  If that is not known, then using the time when the API field changed is acceptable. |
+| **conditions.&#x200b;message** (required) | string | message is a human readable message indicating details about the transition. This may be an empty string. |
```

Comment:
```suggestion
| **conditions.&#x200b;message** (required) | string | a human-readable message indicating details about the transition. This may be an empty string. |
```